tests/containers/zypper_docker.pm

```diff
@@ -7,6 +7,7 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
+# package: zypper-docker
```

Comment:
You didn't... :sweat_smile:

Reply:
If you really want to I can make the check case insensitive... code's not merged yet.

Reply:
Is it not all lower case for `zypper-docker`?